lib/media/ttml_text_parser.js

```diff
@@ -30,11 +30,14 @@ goog.require('shaka.util.StringUtils');
  * @param {number} offset
  * @param {?number} segmentStartTime
  * @param {?number} segmentEndTime
+ * @param {?boolean} useRelativeCueTimestamps
```

Comment:
Drop "?" and add a comment that useRelativeCueTimestamps is only used by the VTT parser.